src/pcrlock/pcrlock.c

```diff
@@ -1956,6 +1956,19 @@ static const char *ansi_true_color(uint8_t r, uint8_t g, uint8_t b, char ret[sta
         return ret;
 }
 
+/* red and green colors usually have good/bad meaning. So exclude color ranges that look too red- or
+ * greenish. See https://en.wikipedia.org/wiki/Hue */
+static double map_pcr_to_hue(uint32_t pcr) {
+        unsigned exclude_red_range = 60;
+        unsigned exclude_green_range = 60;
```

Comment:
hues are floating point values here, i'd thus use "double" rather than "unsigned" too for these values. it's the the choice of least surprise I would say, followed the logic that for hue values we should always use the same type